src/Sulu/Bundle/PageBundle/Markup/Link/ExternalLinkProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Markup\Link;
+
+use Ramsey\Uuid\Uuid;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkConfigurationBuilder;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkItem;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkProviderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ExternalLinkProvider implements LinkProviderInterface
```

Comment:
The builder looks kind of strange here nothing which we can give BC Support at current state so mark it as internal.

```suggestion
/**
 * @internal
 */
class ExternalLinkProvider implements LinkProviderInterface
```

Reply:
move to MarkupBundle